lib/api/v1/controllers/results.go

```diff
@@ -22,7 +23,10 @@ func (c *ResultsController) List(ctx *gin.Context) {
 		return
 	}
 
-	results, err := models.GetUserResults(c.CurrentUser.ID, []string{"User", "AdLinks", "Links"})
+	resultSearchForm := &forms.ResultSearchForm{}
+	_ = ctx.ShouldBindWith(resultSearchForm, binding.JSON)
```

Comment:
We often use query parameters when passing filters on `GET` requests, should we do the same here?
The JSON data is usually used in other requests 🤔

Reply:
Nice catch 🙈 I totally forgot about that, fixed in [ecbf271](https://github.com/carryall/go-google-scraper-challenge/pull/106/commits/ecbf271ad56d1298038e73f4bf7f567f1ca1a59b)